app/controllers/FinancialDetailCreditsController.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import connectors.FinancialDetailsConnector
+import connectors.httpParsers.ChargeHttpParser.UnexpectedChargeResponse
+import controllers.predicates.AuthenticationPredicate
+import models.credits.CreditsModel
+import play.api.Logging
+import play.api.libs.json.Json
+import play.api.mvc._
+import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
+
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.ExecutionContext
+
+@Singleton
+class FinancialDetailCreditsController @Inject()(authentication: AuthenticationPredicate,
+                                                 cc: ControllerComponents,
+                                                 financialDetailsConnector: FinancialDetailsConnector)
+                                                (implicit ec: ExecutionContext) extends BackendController(cc) with Logging {
+
+  def getCreditsAndRefunds(nino: String, from: String, to: String): Action[AnyContent] = {
```

Comment:
its a bit confusing in terms of naming convention, as you are returning CreditModel only, but method name contains "refunds => ..CreditsAndRefunds